themes/default/layouts/page/pricing.html

```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
+            </li>
+            <li class="px-10 py-3" data-faq-label="billing">
+                <h4><a href="#faq-billing" class="text-gray-500">Billing and Support</a></h4>
+            </li>
+        </ul>
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span><a class="link" href="{{ relref . "/docs/intro/console/organizations" }}">Organizations</a></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>1</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>1</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
-                </div>
-            </div>
+        <p class="text-2xl font-bold lg:hidden mt-12 pl-5">Credits and Pricing</p>
+        <ul class="list-none p-0 mx-5 xl:ml-12" data-faq-type="faq-pricing">
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "What is a Pulumi Credit?" "large_header" true) }}
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span><a class="link" href="{{ relref . "/docs/intro/console/project-and-stack-management" }}">Project stacks</a></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Up to 20</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                        A Pulumi Credit represents managing 1 resource for 1 hour. 1 resource represents a
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#custom-resources" }}">custom resource</a>
+                        (e.g., an Amazon S3 bucket), a <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#components" }}">component resource</a>
+                        which is a logical grouping of resources (e.g., an Amazon EKS cluster), or a
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/stack" }}">stack</a> that contains resources
+                        (e.g., dev, test, prod stacks). You use 1 Pulumi Credit to manage each resource for an hour.
+                    </p>
+                    <p>
+                        For example, the cost of managing 1 stack containing 1 S3 bucket and 1 EC2 instance is
+                        3 Pulumi Credits per hour.
+                    </p>
```

Comment:
```suggestion
                        A Pulumi Credit represents managing one resource for one hour. One resource represents a
                        <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#custom-resources" }}">custom resource</a>
                        (e.g., an Amazon S3 bucket), a <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#components" }}">component resource</a>
                        which is a logical grouping of resources (e.g., an Amazon EKS cluster), or a
                        <a class="underline" href="{{ relref . "/docs/intro/concepts/stack" }}">stack</a> that contains resources
                        (e.g., dev, test, prod stacks). You use one Pulumi Credit to manage each resource for an hour.
                    </p>
                    <p>
                        For example, the cost of managing one stack containing one S3 bucket and one EC2 instance is
                        three Pulumi Credits per hour.
                    </p>
```

this is follow-up to @cnunciato's comment